keyboards/planck/keymaps/default/keymap.c

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2015-2017 Jack Humbert
+/* Copyright 2021 Potential Vehicle Owner
```

Comment:
Could you revert the changes to the default keymap here?

Reply:
Yikes! Yeah. I modified the wrong directory!

reverted in 4918ec0